graphql/resolvers.go

```diff
@@ -2628,19 +2628,37 @@ func (r *taskResolver) GeneratedByName(ctx context.Context, obj *restModel.APITa
 }
 
 func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.APITask) (bool, error) {
-	var perfPlugin *plugin.PerfPlugin
-	pRef, err := r.sc.FindProjectById(*obj.ProjectId, false)
+	flags, err := evergreen.GetServiceFlags()
 	if err != nil {
 		return false, err
 	}
-	if perfPluginSettings, exists := evergreen.GetEnvironment().Settings().Plugins[perfPlugin.Name()]; exists {
-		err := mapstructure.Decode(perfPluginSettings, &perfPlugin)
+	if flags.PluginAdminPageDisabled {
+		lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(*obj.ProjectId, -1)
+		if err == nil && lastGoodVersion != nil {
+			parserProject, err := model.ParserProjectFindOneById(lastGoodVersion.Id)
+			if err == nil && parserProject != nil && *parserProject.PerfEnabled {
+				return true, nil
+			}
+		}
+		project, err := model.FindMergedProjectRef(*obj.ProjectId)
```

Comment:
can use FindProjectById as well (although it should use `true` here since we do want the merged one)